Platon/backend/app/auth_system/views.py

```diff
@@ -173,22 +180,32 @@ def get(self):
             # If it fails, an error is raised.
             try:
                 existing_user = User.query.filter_by(id=form.user_id.data).first()
-                user_job = Jobs.query.filter(Jobs.id == existing_user.job_id).first()
             except:
                 return make_response(jsonify({"error" : "The server is not connected to the database."}), 500)
             else:
                 # Checks whether there is an already existing user in the database with the given user ID.
                 # If yes, user information is returned.
                 # If not, an error is raised.
                 if existing_user is not None:
-                    # DO NOT FORGET TO WRITE CODE FOR PRIVATE ACCOUNTS
+                    # Checks whether the requester user follows the requested user or not.
+                    if Follow.query.filter_by(follower_id=requester_id, following_id=form.user_id.data).first() is not None:
+                        following_status = 1 # Represents that the requester user follows the requested user.
+                    elif FollowRequests.query.filter_by(follower_id=requester_id, following_id=form.user_id.data).first() is not None:
+                        following_status = 0 # Represents that the requester user has already sent a follow request to the requested user and it is pending.
+                    else:
+                        following_status = -1 # Represents that the requester user does not follow the requested user and has not yet sent a request to follow them.
+                    
+                    user_job = Jobs.query.filter(Jobs.id == existing_user.job_id).first()
+
                     account_information = { 
                                         "id": existing_user.id,
                                         "name": existing_user.name,
                                         "surname": existing_user.surname,
+                                        "e_mail": existing_user.e_mail,
+                                        "is_private": existing_user.is_private,
+                                        "following_status": following_status,
                                         "rate": existing_user.rate,
                                         "profile_photo": existing_user.profile_photo,
```

Comment:
Why there is no field that represents the institution of the user. As far as I know, in the last Backend meeting we decided to add this part to the user endpoints. Please make the adaquate changes on modes and current code.

Reply:
As I can see, @hilal-demir added this feature in her PR. So it is resolved.